manifests/init.pp

```diff
@@ -247,6 +249,18 @@
     }
   }
 
+  case $::operatingsystem {
+    'Ubuntu': {
+      if $lsbmajdistrelease >= 14 {
+        service { 'pacemaker':
+          ensure    => running,
```

Comment:
Some questions about this resource:

 1. Is it specific to Ubuntu in version 14 and higher? I believe Debian does have `corosync` and `pacemaker` services, for one.
 2. Is it common practice for modules to `ensure => 'running'`? Or should this be configurable?

As a general remark, `'running'` should really be quoted I think.